core/src/main/java/com/segment/analytics/kotlin/core/Analytics.kt

```diff
@@ -121,6 +116,10 @@ open class Analytics protected constructor(
         add(ContextPlugin())
         add(UserInfoPlugin())
 
+        Telemetry.increment(Telemetry.INVOKE_METRIC,
```

Comment:
just realize this. we could change the Telemetry functions to be like the following:
```
    fun increment(metric: String, buildTags: (MutableMap<String, String>) -> Unit) {
        val map = mutableMapOf<String, String>()
        buildTags(map)
    }
```
and then the usage becomes much cleaner
```
        increment("metric") {
            it["error"] = "error"
            it["config"] = "config"
        }
```
same applies to the other methods